### DISCUSSION TITLE
Welcome to Textadept Discussions!

### DISCUSSION THREAD

**Discussion by orbitalquark** [orbitalquark/textadept#4] [General]
Feel free to discuss anything related to Textadept here. The old mailing list is being retired. If you prefer not to use GitHub, you can still e-mail me personally (see README).

**Comment by dajare** [comment]
I got the notification from the "old" mailing list of the new, preferred "Github Discussions" platform for TA chat.

Just to be clear (for dim types like me), could you clarify which of these things you have in mind:

1. we simply add comments in this discussion topic for any items arising for Textadept chat; **or**
2. we start a new "discussion", where before we would have emailed a new "subject line" to the old mailing list.

It's probably `#2` (is my guess!), but I'd like to make sure. Thanks!

**Comment by orbitalquark** [comment]
Yes, please start a new discussion topic. Sorry for the confusion. I'm new at this.

**Comment by slipsnip** [comment]
I like this idea of github discussions.  Thumbs up from me.

**Comment by sigzero** [comment]
Would I open a "discussion" on the Textadept book(s) that you write? I just have questions before purchasing.

**Comment by orbitalquark** [comment]
Yes, you can open a new discussion topic for that.

**Comment by ontap-lab** [comment]
Hi, am using Textadept on big sur on the new mac-mini M1, I can't figure out how to open two windows at the same time ?
I like to open seperate windows so I can compare xml files side by side ?

**Reply by rgieseke** [reply]
You could use split views or force a new instance:

>The GUI version of Textadept is a single-instance application. This means that after you start Textadept, any time you invoke it again (e.g. opening a file from a file browser or command line), the action happens in the original instance. If you want to run separate instances of Textadept, pass the -f or --force command line flag. On Windows, you can create a shortcut to textadept.exe that passes this flag and use that shortcut to run Textadept. On Linux and BSD,

**Reply by ontap-lab** [reply]
Tx for reply , as I said in OP am using a mac mini with big sur, not via command line .
The gui of textadept I find very easy to use but if I can't have two windows open at the same time am afraid I will have to look for an alternative application, once again thanks for replying.

**Comment by orbitalquark** [comment]
Please open a new discussion for questions like this. Thanks!

**Comment by paaguti** [comment]
When I need to display two files side by side, I split the Textadept window
vertically and there you are. If not, editing two files simultaneously with
the buffers and the tabs should be enough. I use the same strategy in
TextAdept and in Emacs. AaMoF, Emacs will allow multiple windows but warn
you when you have more window.

My .2 cents./PA

On Sun, 10 Jan 2021 at 00:22, ontap-lab <notifications@github.com> wrote:

> Tx for reply , as I said in OP am using a mac mini with big sur, not via
> command line .
> The gui of textadept I find very easy to use but if I can't have two
> windows open at the same time am afraid I will have to look for an
> alternative application, once again thanks for replying.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/orbitalquark/textadept/discussions/4#discussioncomment-271823>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZKU4DENQDX3UG6UYJXE6LSZDQKVANCNFSM4UXPZILQ>
> .
>


-- 
Fragen sind nicht da um beantwortet zu werden,
Fragen sind da um gestellt zu werden
Georg Kreisler
